sbi/inference/trainers/npse/npse.py

```diff
@@ -179,11 +179,12 @@ def train(
         training_batch_size: int = 200,
         learning_rate: float = 5e-4,
         validation_fraction: float = 0.1,
-        stop_after_epochs: int = 200,
+        stop_after_epochs: int = 50,
```

Comment:
was there a reason it was 200 before? maybe NPSE needs more time to converge?

Reply:
So beforehand, even with the ema validation loss, the loss was very noisy, and so we increased the `stop_after_epochs` a lot relative to other methods to counteract this variance. By fixing the times where we evaluate the validation loss, we remove a lot of this variance,  so I think it's safe to reduce `stop_after_epochs`. Note also that now evaluating the validation loss can be quite a bit slower because we evaluate the loss **for each pair in the validation set for each time in `validation_times`**.  So, having a really high stopping threshold can make training quite a bit slower.